Packs/Okta_v2/TestPlaybooks/playbook-OktaV2-Test.yml

```
@@ -1329,37 +1100,114 @@ tasks:
-    view: |-
-      {
-        "position": {
-          "x": 695,
-          "y": 2470
-        }
-      }
+    view: "{\n  \"position\": {\n    \"x\": 1452.5,\n    \"y\": 2995\n  }\n}"
     note: false
     timertriggers: []
     ignoreworker: false
     skipunavailable: false
     quietmode: 0
-view: |-
-  {
-    "linkLabelsPosition": {},
-    "paper": {
-      "dimensions": {
-        "height": 3010,
-        "width": 1885,
-        "x": 50,
-        "y": 50
-      }
-    }
-  }
+  '54':
+    id: '54'
+    taskid: bbdc4cf5-914f-4169-8777-3e0a04cb566c
+    type: regular
+    task:
+      id: bbdc4cf5-914f-4169-8777-3e0a04cb566c
+      version: -1
+      name: Check if Test1@test.com exists
+      description: Searches for Okta users.
+      script: Okta v2|||okta-search
+      type: regular
+      iscommand: true
+      brand: Okta v2
+    nexttasks:
+      '#none#':
+      - '55'
+    scriptarguments:
+      limit: {}
+      term:
+        simple: Test1@test.com
+      verbose: {}
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 377.5,\n    \"y\": 370\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '55':
+    id: '55'
+    taskid: b2820523-8a07-43b3-8493-c6952f7b11ce
+    type: condition
+    task:
+      id: b2820523-8a07-43b3-8493-c6952f7b11ce
+      version: -1
+      name: verify result
+      type: condition
+      iscommand: false
+      brand: ''
+      description: ''
+    nexttasks:
+      '#default#':
+      - '5'
+      yes:
+      - '56'
+    separatecontext: false
+    conditions:
+    - label: yes
+      condition:
+      - - operator: containsString
+          left:
+            value:
+              simple: Account.Email
+            iscontext: true
+          right:
+            value:
+              simple: Test1@test.com
+    view: "{\n  \"position\": {\n    \"x\": 377.5,\n    \"y\": 545\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '56':
+    id: '56'
+    taskid: 347a47ca-000a-43ff-8b21-07a4c9851268
+    type: regular
+    task:
+      id: 347a47ca-000a-43ff-8b21-07a4c9851268
+      version: -1
+      name: okta-delete-user
+      description: Delete specified user
+      script: Okta v2|||okta-delete-user
+      type: regular
+      iscommand: true
+      brand: Okta v2
+    nexttasks:
+      '#none#':
+      - '5'
+    scriptarguments:
+      userId: {}
+      username:
+        simple: Test1@test.com
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 490,\n    \"y\": 720\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+view: "{\n  \"linkLabelsPosition\": {},\n  \"paper\": {\n    \"dimensions\": {\n \
+  \     \"height\": 3505,\n      \"width\": 1782.5,\n      \"x\": 50,\n      \"y\"\
+  : 50\n    }\n  }\n}"
 inputs: []
 outputs: []
+fromversion: 1.0.0
```

it's not tested on previous versions
```suggestion
fromversion: 4.1.0
```